tests/scipy_stats_test.py

```diff
@@ -623,6 +623,38 @@ def testMultivariateNormalLogpdfBatch(self, ndim, nbatch, dtype):
     result2 = jax.vmap(lsp_stats.multivariate_normal.logpdf)(x, mean, cov)
     self.assertArraysEqual(result1, result2)
 
+  @parameterized.named_parameters(jtu.cases_from_list(  {"testcase_name":
+           "_{}_a_shape={}_axis={}_keepdims={}".format(
+             op, jtu.format_shape_dtype_string(a_shape, a_dtype),
+             axis, keepdims),
+         "op": op, "a_shape": a_shape, "a_dtype": a_dtype,
+         "axis": axis,
+         "keepdims": keepdims}
+        for a_dtype in jtu.dtypes.floating
+        for a_shape, axis in (
+          ((7,), None),
+          ((47, 7), 0),
+          ((4, 101), 1),
+        )
+        for keepdims in [False, True]
+        for op in ["mode", "nanmode"]))
```

Comment:
I'm confused by the use of `nanmode` here. In similar tests within `lax_numpy_test.py`, we are testing the functions e.g. `jnp.mean` and `jnp.nanmean`. I don't think there's a `nanmode` function though?